tests/3/expected.json

```diff
@@ -188,8 +188,7 @@
     }
   },
   "LOOPBACK_INTERFACE": {
-    "Loopback0": {},
-    "Loopback0|10.0.0.2/32": {}
+    "Loopback0": {}
```

Comment:
I see no use case for an unnumbered Loopback interface.

Reply:
So omit the whole `LOOPBACK_INTEFACE` object, when no loopback address is given?